core/embed/trezorhal/se_thd89.c

```
@@ -2296,3 +2319,81 @@
+  memcpy(cred_id.rp_id_hash, src, len);
+  return se_set_fido2_data(index * FIDO2_RESIDENT_CREDENTIALS_SIZE,
+                           (uint8_t *)&cred_id, 6 + len);
+}
+
+secbool se_delete_fido2_resident_credentials(uint32_t index) {
+  uint8_t buffer[FIDO2_RESIDENT_CREDENTIALS_HEADER_LEN] = {0xff};
+  return se_set_fido2_data(index * FIDO2_RESIDENT_CREDENTIALS_SIZE, buffer,
+                           FIDO2_RESIDENT_CREDENTIALS_HEADER_LEN);
+}
+
+secbool se_delete_all_fido2_credentials(void) {
```

cppcheck (2.10-2)

[style] The function 'se_delete_all_fido2_credentials' is never used.

(unusedFunction)